KDIC/KdicPlayer.swift

```diff
@@ -0,0 +1,40 @@
+//
+//  KdicPlayer.swift
+//  KDIC
+//
+//  Created by Shaun Mataire on 11/9/15.
+//  Copyright © 2015 AppDev. All rights reserved.
+//
+
+import Foundation
+import AVFoundation
+
+class KdicPlayer {
+    static let sharedInstance = KdicPlayer()
+    
+    private var player = AVPlayer(URL: NSURL(string: "http://kdic.grinnell.edu:8001/stream")!)
+    private var isPlaying = false
```

Comment:
Use `public private(set) var isPlaying = false` and you can get rid of `currentlyPlaying()` from line 36.